### ISSUE TITLE
marvel fix and docker image available

### ISSUE THREAD

**Issue by papanikos** [replikation/What_the_Phage#133]
Hi @mult1fractal @replikation ,

I was running into an issue with marvel failing on some input. 
* The main issue is described on papanikos/MARVEL#1
* A PR is awaiting approval [here](https://github.com/LaboratorioBioinformatica/MARVEL/pull/6) - the author has been contacted and seemed to agree.

In the meantime:
* A new [Dockerfile](https://github.com/papanikos/What_the_Phage/blob/marvel_fix/phage-tool-Dockerfiles/MARVEL/Dockerfile) and its respective [container](https://hub.docker.com/r/papanikos/marvel/tags?page=1&ordering=last_updated) are available.
* A `-profile smalltest,local,singularity` run worked ok.

Shall I open a new PR for this? Would you prefer having things first merged upstream and then deciding?

**Comment by replikation** [comment]
hi, thanks for reporting. This would be basically fixed just by using a newer docker container if I understand this correctly? (the moment marvel releases a new version)

**Comment by papanikos** [comment]
Yes, that is correct. This is what I did and reporting here - a container with the patch included, available as `papanikos/marvel:0.2-e2591bb`.

What I put in the container is the branch with the fix from my fork - whereas I think it would be better to just use one entry point in general. In this case, that would be the original repo, properly updated etc.

I will just create a new PR that updates the Dockerfile and the container used, and you can decide if you want to merge it or not. I can always keep using my `container.config`.

---
_Incoming mini-rant but I need to vent_ :smiley:  :

The notion described in this statement 

>(the moment marvel releases a new version)

just seems so 'out there' when it comes to these tools. The vast majority are just abandoned. No effort into properly packaging anything, let alone maintaining. Just publish a manuscript, put some script on Github and move on. That is fine and even understandable - the show must go on. 

It becomes a very annoying issue, though, when you actually want to do something more than 

```
$ python run_this_with_the_exact_same_setup_I_used_when_I_copied_it_from_my_jupyter_notebook.py
```

Containers are nice, but if it takes me 5min to figure out what is wrong with the original and 5 days - exaggerated - to build a container just trying to handle software and data dependencies by applying patches at build time (see [this for example](https://github.com/replikation/What_the_Phage/blob/da4e9f539bb7074952578957710acf4f35479dbb/phage-tool-Dockerfiles/MARVEL/Dockerfile#L14))... it is plain frustrating.

**Comment by replikation** [comment]
yep, i totally agree on this, one of the reasons we couldn't "just" use conda was that we needed to "adjust" a few tools as the installation was a nightmare followed up by the execution.
